theme/components/balloonpanel.scss

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+// For licensing, see LICENSE.md or http://ckeditor.com/license
+
+$ck-balloon-arrow-offset: 2px;
+$ck-balloon-arrow-height: 15px;
+$ck-balloon-arrow-half-width: 10px;
+
+.ck-balloon-panel {
+	@include ck-unselectable();
+	@include ck-rounded-corners();
+
+	min-width: 50px;
+	min-height: 15px;
+
+	display: none;
+	position: absolute;
+	background: ck-color( 'background' );
+	border: 1px solid ck-border-color( 'foreground' );
+
+	// TODO: Create helper for shadows.
+	box-shadow: 0 0 5px ck-color( 'foreground', -15 );
```

Comment:
Let's create a mixin for all floating panels, toolbars, etc. in `helpers/_shdadow.scss`

```scss
@mixin ck-shadow() {
    box-shadow: 0 0 5px ck-color( 'foreground', -15 );
}
```

and use it here (after `@include ck-rounded-corners();`) and in the [dropdown](https://github.com/ckeditor/ckeditor5-theme-lark/blob/t/33/theme/components/dropdown.scss#L33). Then let's wait for more use cases to decided if it should be configurable, etc.